awx/main/utils/common.py

```diff
@@ -1079,3 +1082,38 @@ def truncate_stdout(stdout, size):
             set_count += 1
 
     return stdout + u'\u001b[0m' * (set_count - reset_count)
+
+def create_partition(tblname, start=None, end=None, partition_label=None, minutely=False):
+    """Creates new partition table for events.
+    - start defaults to beginning of current hour
+    - end defaults to end of current hour
+    - partition_label defaults to YYYYMMDD_HH
+
+    - minutely will create partitions that span _a single minute_ for testing purposes
```

Comment:
This is neat and helpful for testing; in the final version of this PR prior to merge, though, let's remove the per-minute support (having this accidentally called would be bad, because postgres isn't going to like having this many partitions over time).